### DISCUSSION TITLE
Getting Exit Code 9?

### DISCUSSION THREAD

**Discussion by Joyari09** [qnblackcat/uYouPlus#1544] [General]
Make sure you enter a download link. I used gdrive for where I uploaded my IPA and made the mistake of only using the view link from the anyone with the link can view thing and got the error. What I did is download the IPA I had in my drive and I went to my download history on chrome, once it finished downloading, I right clicked on the name of the file and copied the link address. That's the download link that you should add on the Run Workflow. One more thing, Make sure your IPA is named exactly "YouTube.ipa", I think it's case sensitive, you might get the Error Exit Code 1 if you don't.

**Comment by dreamless17** [comment]
renaming to "YouTube.ipa" helped， thanks
